test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/RejectCaseControllerSpec.scala

```diff
@@ -214,7 +214,7 @@ class RejectCaseControllerSpec extends WordSpec with Matchers with UnitSpec
   "View Confirm page for a rejected case" should {
 
     "return OK and HTML content type" in {
-      when(casesService.suspendCase(refEq(caseWithStatusREJECTED), refEq(operator))(any[HeaderCarrier])).thenReturn(successful(caseWithStatusREJECTED))
+      when(casesService.rejectCase(refEq(caseWithStatusREJECTED), refEq(operator))(any[HeaderCarrier])).thenReturn(successful(caseWithStatusREJECTED))
```

Comment:
Will need (, any[FileUpload], any[String]) passing into rejectCase call, but guess you'll have already spotted as it won't compile

Reply:
Forgot about this, sorted